app/services/merchant_service.rb

```diff
@@ -0,0 +1,25 @@
+class MerchantService
+  def self.merchants
+    content = conn.get("/api/v1/merchants")
+    merchants = (parse_response(content))[:data]
+    merchants.map do |merchant|
+      {name: merchant[:attributes][:name], id: merchant[:id]}
+    end
+  end
+
+  def self.merch_items(id)
+    content = conn.get("/api/v1/merchants/#{id}/items")
+    items = (parse_response(content))[:data]
+    items.map do |item|
+      {name: item[:attributes][:name], id: item[:id]}
+    end
+  end
```

Comment:
Nice job with this! This goes for all of these methods here, but also create POROs to send back to your controller. You could have a Merchant, Item, and MerchantItem poros.